plugins/spamassassin.js

```diff
@@ -320,18 +320,18 @@ exports.msg_too_big = function (connection) {
 
     const max = plugin.cfg.main.max_size;
     if (size <= max) { return false; }
-    connection.loginfo(plugin, 'skipping, size ' + utils.prettySize(size) +
-            ' exceeds max: ' + utils.prettySize(max));
+    connection.loginfo(plugin, `skipping, size ${utils.prettySize(size)}` +
+            ` exceeds max: ${utils.prettySize(max)}`);
     return true;
 };
 
 exports.log_results = function (connection, spamd_response) {
     const plugin = this;
     const cfg = plugin.cfg.main;
-    connection.loginfo(plugin, "status=" + spamd_response.flag +
-          ', score=' + spamd_response.score +
-          ', required=' + spamd_response.reqd +
-          ', reject=' + ((connection.relaying) ?
-            (cfg.relay_reject_threshold || cfg.reject_threshold) : cfg.reject_threshold) +
-          ', tests="' + spamd_response.tests + '"');
+    connection.loginfo(plugin, `status=${spamd_response.flag}` +
+          `, score=${spamd_response.score}` +
+          `, required=${spamd_response.reqd}` +
+          `, reject=${((connection.relaying) ?
+            (cfg.relay_reject_threshold || cfg.reject_threshold) : cfg.reject_threshold)}` +
```

Comment:
whitespace lint error here (see Travis CI details or `npm run lint` locally. I'd suggest just including this on the previous line. I think since the adoption of eslint (and deprcation of jslint), we are (at least mostly) past worrying about long lines.